src/Lean/ToLevel.lean

```diff
@@ -0,0 +1,44 @@
+/-
+Copyright (c) 2023 Kyle Miller. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kyle Miller, Alex Keizer
+-/
+prelude
+import Lean.Expr
+
+/-! # `ToLevel` class
+This module defines `Lean.ToLevel`, which is the `Lean.Level` analogue to `Lean.ToExpr`.
+-/
+
+namespace Lean
+
+universe w
+
+/-- A class to create `Level` expressions that denote particular universe levels in Lean.
+`Lean.ToLevel.toLevel.{u}` evaluates to a `Lean.Level` term representing `u` -/
+class ToLevel.{u} : Type where
+  /-- A `Level` that represents the universe level `u`. -/
+  toLevel : Level
+  /-- The universe itself. This is only here to avoid the "unused universe parameter" error.
+    We'll remove this field once https://github.com/leanprover/lean4/issues/2116 gets fixed.
+  -/
```

Comment:
Some tweaks to emphasize that it's a hack and to deemphasize any promises that it'll ever be removed (I hope it does, but I haven't heard any motion on 2116 one way or another yet)
```suggestion
  /-- A hack to avoid the "unused universe parameter" error.
    We can remove this field pending issue https://github.com/leanprover/lean4/issues/2116 -/
```